lightningd/coin_mvts.c

```diff
@@ -1,24 +1,61 @@
 #include <lightningd/coin_mvts.h>
 #include <lightningd/notification.h>
 
+static s64 update_count(struct lightningd *ld)
+{
+	s64 count;
+	bool db_in_tx;
+	db_in_tx = db_in_transaction(ld->wallet->db);
```

Comment:
I'm curious about what callchains do this outside a transaction?  That has a weird smell to it...

Reply:
Hm I can't seem to find them. I'm going to push the non-switched version up and see if Travis can find them for me :)